src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { SearchDiv, SearchTextInput, SearchInput } from "./styles";
+import fetchRecipes from "../../api/fetchRecipes";
+
+const placeholderCaloriesLimit = 2000;
+
+const placeholderIngredientList = "Chicken salmon cream cheese";
+
+const search = async (ingredient = "", caloriesLimit = 0) => {
```

Comment:
And here use it like

`const search = async ({ ingredient: "", caloriesLimit: 0 }) => `